build/configure.js

```diff
@@ -3,13 +3,17 @@
 'use strict';
 
 var clone = require('clone');
+var dot = require('@deque/dot');
 var templates = require('./templates');
 var buildManual = require('./build-manual');
 var entities = new (require('html-entities')).AllHtmlEntities();
+var dotRegex = /\{\{.+?\}\}/g;
```

Comment:
It makes the `.` not greedy. So instead of `{{foo bar}} {{foo}}` matching the entire string, it will stop at the first set of `}}` so you get 2 matches.

Reply:
ahh, didn't know that. Cool